src/drawer.multicanvas.js

```diff
@@ -332,11 +332,15 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
 
     getImage: function(type, quality) {
         // combine all available canvasses together
-        var availableCanvas = '';
+        var availableCanvas = [];
         for (var i in this.canvases) {
             var getEntry = this.canvases[i].wave.getContext('2d');
-            availableCanvas += getEntry.canvas.toDataURL(type, quality);
+            availableCanvas.push(getEntry.canvas.toDataURL(type, quality));
+        }
+        if (this.canvases.length == 1) {
+            return availableCanvas[0];
+        } else {
+            return availableCanvas;
```

Comment:
It's a matter of taste, but I'd prefer if you did `return availableCanvas.length > 1 ? availableCanvas : availableCanvas[0];`